providers/tentacle.rb

```diff
@@ -94,7 +94,7 @@
     action :run
     cwd tentacle_install_location
     code <<-EOH
-      .\\Tentacle.exe new-certificate -e "#{cert_file}" --console
```

Comment:
Would `--if-blank` do the same thing? That would not overwrite the cert if it exists